[2024-12-23T05:52:05.028+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-23T05:52:05.037+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: call_9851100067_2024-12-23T05-51-45-325611Z.wait_until_call scheduled__2024-12-23T05:51:45.325611+00:00 [queued]>
[2024-12-23T05:52:05.040+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: call_9851100067_2024-12-23T05-51-45-325611Z.wait_until_call scheduled__2024-12-23T05:51:45.325611+00:00 [queued]>
[2024-12-23T05:52:05.040+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2024-12-23T05:52:05.045+0000] {taskinstance.py:2889} INFO - Executing <Task(_PythonDecoratedOperator): wait_until_call> on 2024-12-23 05:51:45.325611+00:00
[2024-12-23T05:52:05.050+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=1704) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-12-23T05:52:05.051+0000] {standard_task_runner.py:72} INFO - Started process 1705 to run task
[2024-12-23T05:52:05.052+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'call_9851100067_2024-12-23T05-51-45-325611Z', 'wait_until_call', 'scheduled__2024-12-23T05:51:45.325611+00:00', '--job-id', '892', '--raw', '--subdir', 'DAGS_FOLDER/complete_dag.py', '--cfg-path', '/tmp/tmp0bx7nzkg']
[2024-12-23T05:52:05.052+0000] {standard_task_runner.py:105} INFO - Job 892: Subtask wait_until_call
[2024-12-23T05:52:05.074+0000] {task_command.py:467} INFO - Running <TaskInstance: call_9851100067_2024-12-23T05-51-45-325611Z.wait_until_call scheduled__2024-12-23T05:51:45.325611+00:00 [running]> on host 1868de310944
[2024-12-23T05:52:05.111+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='call_9851100067_2024-12-23T05-51-45-325611Z' AIRFLOW_CTX_TASK_ID='wait_until_call' AIRFLOW_CTX_EXECUTION_DATE='2024-12-23T05:51:45.325611+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-12-23T05:51:45.325611+00:00'
[2024-12-23T05:52:05.111+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-23T05:52:05.113+0000] {logging_mixin.py:190} INFO - 199
[2024-12-23T05:54:05.113+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-12-23T05:54:05.119+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-23T05:54:05.120+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=call_9851100067_2024-12-23T05-51-45-325611Z, task_id=wait_until_call, run_id=scheduled__2024-12-23T05:51:45.325611+00:00, execution_date=20241223T055145, start_date=20241223T055205, end_date=20241223T055405
[2024-12-23T05:54:05.169+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-12-23T05:54:05.182+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-12-23T05:54:05.182+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
